src/System.Management.Automation/engine/runtime/CompiledScriptBlock.cs

```diff
@@ -2191,36 +2176,14 @@ internal static void LogScriptBlockEnd(ScriptBlock scriptBlock, Guid runspaceId)
         internal bool HasCleanBlock { get => AstInternal.Body.CleanBlock != null; }
     }
 
-    [Serializable]
-    internal class ScriptBlockSerializationHelper : ISerializable, IObjectReference
+    internal class ScriptBlockSerializationHelper
```

Comment:
This class can be removed now, isn't it?

Reply:
removed